WEB-559 #selector jumps below title

fladens opened the pull request:
On the homepage when adding the #selector to the URL it should jump
exactly to the top of the selector title.

Added a :target css to index.scss so that the title has a top offset
Also added a scroll to the title to the Main.js, in case that the
content loads after the jump.

dmetzner commented:
I tested the implementation for following link: **pocketcode/#mostDownloaded**

- I think the desired representation would be the one that can be seen here
![correct](https://user-images.githubusercontent.com/40868718/47523462-60797000-d898-11e8-9c91-a5440d3c38f0.png)
 I only get this view if i re-enter the link.

- When i refresh a page or load it for the first time i get following representation:
![incorrect](https://user-images.githubusercontent.com/40868718/47523678-e09fd580-d898-11e8-8bf0-34d7f9f783ec.png)
 Seems a bit **inconsistent**?

- Last but not least, please use **variables instead of hardcoded values**. This allows an easy way to adjust things for differerent themes. not all headers have the same size. (E.g. create@school)
![theme](https://user-images.githubusercontent.com/40868718/47523919-86ebdb00-d899-11e8-8772-ba396fba4233.png)

fladens committed:
WEB-559 #selector jumps below title

On the homepage when adding the #selector to the URL it should jump
exactly to the top of the selector title.

Added a javascript to jump to the title with offset of the
navigationbar and also to jump after program loading is finished.

fladens commented:
@dmetzner 
I changed it to javascript function so it waits for the page to be loaded and then it calculates the height and jumps to the title.
Added load, hashchange and immediate jump.
